lib/srthelper.rb

```diff
@@ -0,0 +1,141 @@
+
+require "json"
+
+SENTENCE_CONTINUATION_DIFF = 0.1
+SINGLE_SRT_SECONDS_LENGTH = 3
+CONTINUATION_SENTENCE_OVERLOAD = 3
+
+
+class SrtHelper
+
+	def parse_file(input_file, output_file)
+		cf =File.read(input_file)
```

Comment:
We should revisit this logic again.. What if the input asset is an hour long and then we might end up generating a really big transcribe output which we are attempting to load as JSON in-memory. See if there are any streaming way of json parsing available !!

Reply:
Will test with big source file and share the test result.